### PR TITLE
feat: support create E2E and IT case from Einstein

### PR DESCRIPTION
**Pull request by chenxq**

### CODE REVIEW

File: src/commands/tees-create.command.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  `${extensionNamespace}.teesCreate`,
+  async () => {
+    const caseId = await vscode.window.showInputBox({
+      placeHolder: 'Please enter case ID or IDs(eg. 1000 or 1000, 1001)',
+    })
+    if (caseId) {
+      const { execCommand, caseType, rootPath } = await generateCreateCommand(caseId);
+      await vscode.window.showInformationMessage(`Creating ${caseType} case RCI-${caseId}......`);
+
+      execCreateCommand(rootPath, execCommand);
```
Comment (ZouYouShun) [inline src/commands/tees-create.command.ts]: `generateCreateCommand` and `execCreateCommand` do not need to be separated into two files, and those methods do not need to become two steps, 


create a class called `TestGenerator`, and have some method in this class

```
export class TestGenerator {
    downloadTeesFromEinstein(){
        ... // all logic in here, also can separate to lot of method, to make call readable.   
    }
}
```

that will be helpful when we also want to create another type of test file in the future.

File: src/utils/tees/generateCreateCommand.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import * as fs from 'fs';
+import * as vscode from 'vscode';
+
+export async function generateCreateCommand(caseId: string) {
+    let caseType, projectPath;
+    let projectList;
+    const { rootPath } = vscode.workspace;
+    if (rootPath) {
+        fs.readdir(`${rootPath}/packages/`, (err, files) => {
```
Comment (ZouYouShun) [inline src/utils/tees/generateCreateCommand.ts]: in cli, we can use `fs.readdirSync` is ok

File: src/utils/tees/generateCreateCommand.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+            projectList = files;
+        });
+    }
+    if (caseId) {
```
Comment (ZouYouShun) [inline src/utils/tees/generateCreateCommand.ts]: should check caseId first, if not case id should break outside

File: src/utils/tees/generateCreateCommand.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        try {
+            caseType = await vscode.window.showQuickPick(
+                ['E2E', 'UT/IT'], { canPickMany: false }
+            );
```
Comment (ZouYouShun) [inline src/utils/tees/generateCreateCommand.ts]: just check here, if no value break

File: src/utils/tees/generateCreateCommand.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+                    projectList, { canPickMany: false }
+                );
+            } catch (error) {
+                vscode.window.showErrorMessage(error);
+            }
+        }
+    }
+
+
+    const execCommand = caseType === 'E2E' ? `yarn run tees create ${caseId}` : `CASE_TYPE=IT PROJECT_PATH=${projectPath} yarn run tees create ${caseId}`;
```
Comment (ZouYouShun) [inline src/utils/tees/generateCreateCommand.ts]: `yarn run tees create ${caseId}` seems is the repeat part, here should

```ts
const prefixCommand = caseType === 'E2E'? '' : `CASE_TYPE=IT PROJECT_PATH=${projectPath} `;
const execCommand = `${prefixCommand}yarn run tees create ${caseId}`
```

File: src/utils/tees/teesCreateCase.ts [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+    if (caseType === 'UT/IT' && projectList) {
+      projectPath = await vscode.window.showQuickPick(projectList, {
+        canPickMany: false,
+      });
+    }
+  }
+
+  const execCommand =
+    caseType === 'E2E'
+      ? `yarn run tees create ${caseId}`
```
Comment (ZouYouShun) [inline src/utils/tees/teesCreateCase.ts]: `yarn run tees create ${caseId}` is deprecated part

File: src/utils/tees/teesCreateCase.ts [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+    const _dir = `${rootPath}/packages/`;
+    if (fs.existsSync(_dir)) {
+      projectList = fs.readdirSync(_dir);
+    }
+  }
+  if (caseId) {
+    caseType = await vscode.window.showQuickPick(['E2E', 'UT/IT'], {
+      canPickMany: false,
+    });
+    if (caseType === 'UT/IT' && projectList) {
```
Comment (ZouYouShun) [inline src/utils/tees/teesCreateCase.ts]: why this if in the caseId if

File: src/utils/tees/teesCreateCase.ts [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+import * as vscode from 'vscode';
+import { OutputChannel } from '..';
+
+export async function teesCreateCase() {
+  const caseId = await vscode.window.showInputBox({
+    placeHolder: 'Please enter case ID or IDs(eg. 1000 or 1000, 1001)',
+  });
+
+  let caseType, projectPath;
+  let projectList;
```
Comment (ZouYouShun) [inline src/utils/tees/teesCreateCase.ts]: should announce type

File: src/commands/tees-create.command.ts (hunk 1 above)
Comment (chenxq) [reply src/commands/tees-create.command.ts]: fixed

File: src/utils/tees/teesCreateCase.ts (hunk 7 above)
Comment (chenxq) [reply src/utils/tees/teesCreateCase.ts]: because if no caseId input, no need to select project

**Pull request merged**